src/viam/api/CMakeLists.txt

```diff
@@ -287,6 +293,7 @@ target_sources(viamapi
     ${PROTO_GEN_DIR}/google/api/annotations.pb.cc
     ${PROTO_GEN_DIR}/google/api/http.pb.cc
     ${PROTO_GEN_DIR}/google/api/httpbody.pb.cc
+    ${PROTO_GEN_DIR}/google/rpc/status.pb.cc
```

Comment:
⬇️ Very likely you also need to add `status.pb.h` down below line 340.

Reply:
I agree that it looks like it should be required. I was able to build without that line, but it feels more proper to add it anyway. I've added it. Things continue to build when using a local copy of the protos, and I still don't know what the plan is to update the ones in this repo itself.

To rebuild things, I've been using this: `cd ..; rm -rf build; mkdir build; cd build; cmake -DVIAMCPPSDK_OFFLINE_PROTO_GENERATION=ON -DVIAMCPPSDK_USE_DYNAMIC_PROTOS=ON -DVIAMCPPSDK_USE_LOCAL_PROTOS=ON .. -G Ninja; ninja -j 5 all` and I imagine that the final version will use fewer options on the `cmake` part.

Reply:
You would only notice it to be missing if you tried to build a program against an installed build of the SDK.